src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+SUDO_CMD=sudo
+
+# Parse script argument that allows users to customize sudo command
+for i in "$@"
+do
+    case "$i" in
+        --sudo-cmd=*)
+        SUDO_CMD="${i#*=}"
+        shift # past argument=value
+        ;;
+    esac
+done
+
+install_shared_packages() {
+    pkg_manager=$1
+    install_verb=$2
+
+    local shared_packages="git curl apt-transport-https"
+    for package in $shared_packages
+    do
+        if [ $pkg_manager = apk ]; then
+            $SUDO_CMD $pkg_manager $install_verb $package
+        else
+            $SUDO_CMD $pkg_manager $install_verb $package -y
+        fi
+    done
+}
+
+download_dotnet_script() {
+    curl -o dotnet-install.sh -L https://dot.net/v1/dotnet-install.sh
```

Comment:
This `curl` call could fail, but the script would not stop in that case. How about calling `set -e` at the beginning of this script, so that the script execution stops as soon as a command failed?

Reply:
..or a `|| exit 1` or `die(..)` even.